lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +191,38 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.config = config
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        article_href = article_bs.get('href')
+        if isinstance(article_href, str) and article_href.startswith('/doc/'):
+            return f'https://www.kommersant.ru{article_href}'
```

Comment:
is there a way to not hardcode the url string?

Reply:
will it be ok to create separate variable inside this function which contains `'https://www.kommersant.ru'` and then return sequence of this variable and article_href?

Reply:
instead you can try to get this link dynamically from each url extracted or get this link dynamically from seed_urls at least